### DISCUSSION TITLE
Supporting Channels

### DISCUSSION THREAD

**Discussion by denelon** [microsoft/winget-cli#1445] [Q&A]
When we initially thought about channels, we were looking at Docker Desktop as an example. It doesn't support having both stable and beta releases side by side. In order to do the right thing with upgrade, we need to know what channel a user is on. Several other applications fall into the same category. 

I'd like to see if we can generate a list of packages that work like this so we can start rounding out what support for channels would look like.

**Comment by jedieaston** [comment]
Office uses a channel setup, although it’s configured by the arguments passed to ODT instead of by a different installer, so I don’t know if it would work with the current design for winget’s channels. 

https://docs.microsoft.com/en-us/deployoffice/overview-update-channels lists the prod ones, plus there’s at least one Insider branch.
